ParcelDistributionCenter/ParcelDistributionCenter.Web/Views/Shared/_Layout.cshtml

```diff
@@ -2,64 +2,88 @@
 <html lang="en">
 <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="" />
+    <meta name="author" content="" />
     <title>@ViewData["Title"] - ParcelDistributionCenter.Web</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
-    <link rel="stylesheet" href="~/ParcelDistributionCenter.Web.styles.css" asp-append-version="true" />
+    <link href="https://cdn.jsdelivr.net/npm/simple-datatables@latest/dist/style.css" rel="stylesheet" />
```

Comment:
Czemu się tutaj zdecydowałeś na wpisanie jakiegoś URLa zamiast tych styleSheetów co powyżej? Ja taki biegły w tym jeszcze nie jestem i się zastanawiam skąd to wziąłeś i ska wiesz jak tego używać :P

Reply:
To też jest kopia z szablonu...